src/test/java/org/hyperledger/fabric/sdkintegration/NetworkConfigIT.java

```diff
@@ -169,6 +171,27 @@ public static void doMainSetup() throws Exception {
         mockuser.setEnrollment(hfcaClient.enroll(mockuser.getName(), mockuser.getEnrollmentSecret()));
         orgRegisteredUsers.put(org.getName(), mockuser);
 
+//        if (testConfig.isRunningFabricTLS()) {
```

Comment:
This line isn't commented out in your cherry-pick to _release-2.2_, and that build looks to have passed

Reply:
Please disregard the two pull requests to master and release-2.2 for now. Once they are ready I will move them from draft status to ready to review. We are currently having trouble with the integration test

Reply:
The PR is ready for code review